src/DDLQueryBuilder.php

```diff
@@ -223,11 +237,8 @@ public function getColumnDefinition(string $table, string $column): string
         $result = '';
         $quotedTable = $this->quoter->quoteTableName($table);
 
-        /** @var QueryBuilderPDOMysql $qb */
-        $qb = $this->queryBuilder;
-
         /** @var array<array-key, string> $row */
-        $row = $qb->command()->setSql('SHOW CREATE TABLE ' . $quotedTable)->queryOne();
+        $row = $this->command->setSql('SHOW CREATE TABLE ' . $quotedTable)->queryOne();
```

Comment:
use this $this->schema->getCreateTableSql()
and remove $this->command from class